src/datetimepicker.windows.js

```diff
@@ -50,13 +51,26 @@ export default class DateTimePickerWindows extends React.Component<WindowsNative
       timezoneOffsetInSeconds = -60 * this.props.value.getTimezoneOffset();
     }
 
-    return (
-      <RNDateTimePickerWindows
-        {...props}
-        onChange={this._onChange}
-        timeZoneOffsetInSeconds={timezoneOffsetInSeconds}
-      />
-    );
+    // 'date' is the default mode
+    if (this.props.mode === 'date' || this.props.mode === undefined) {
```

Comment:
would you please rewrite this to use function component (with hooks) and also consider the case when `mode === null` (should be handled the same as `undefined` I think). Thanks!

Reply:
Rewrote as function component. 
Handled both undefined and null using the following comparison:
"mode **==** null"